modules/client-configuration/pages/system-set-manager.adoc

```diff
@@ -0,0 +1,558 @@
+[[ssm]]
+= System Set Manager
+
+The System Set Manager (SSM) is used to perform actions on more than one client at a time.
+SSM creates ephemeral sets of clients, making it useful for one-off actions that you need to apply to a number of clients.
+If you want more permanent sets, consider using system groups instead.
+For more information about system groups, see xref:client-configuration:system-groups.adoc[].
+
+
+The actions available for use in SSM are listed in this table.
+The icons in this table indicate:
+
+* {check} this action is available in SSM for this client type
+* {cross} this action is not available in SSM for this client type
+* {question} this action is under consideration for this client type, and may or may not be supported at a later date.
+
+.Available SSM Actions
+[cols="1,1,1", options="header"]
+|===
+| Action                    | Traditional   | Salt Clients
+| List Systems              | {check}       | {check}
+| Install Patches           | {check}       | {check}
+| Schedule Patch Updates    | {check}       | {check}
+| Upgrade Packages          | {check}       | {check}
+| Install Packages          | {check}       | {check}
+| Remove Packages           | {check}       | {check}
+| Verify Packages           | {check}       | {cross}
+| Create Groups             | {check}       | {check}
+| Manage Groups             | {check}       | {check}
+| Channel Memberships       | {check}       | {check}
+| Channel Subscriptions     | {check}       | {cross}
+| Deploy/Diff Channels      | {check}       | {cross}
+| Autoinstall Clients       | {check}       | {cross}
+| Tag for Snapshot          | {check}       | {cross}
+| Remote Commands           | {check}       | {cross}
+| Power Management          | {check}       | {cross}
+| Update System Preferences | {check}       | {check}
+| Update Hardware Profiles  | {check}       | {check}
+| Update Package Profiles   | {check}       | {check}
+| Set/Remove Custom Values  | {check}       | {check}
+| Reboot Clients            | {check}       | {check}
+| Migrate clients to another Organization | {check} | {check}
+| Delete clients            | {check} | {check}
```

Comment:
```suggestion
| Delete Clients            | {check} | {check}
```

Reply:
Hrm, I see why you're doing this. Do you think it might be better to switch everything to sentence case, though?